### DISCUSSION TITLE
Javascript for averaging row data??

### DISCUSSION THREAD

**Discussion by drearystate** [Budibase/budibase#8772] [Help]
Hello, I'm pretty new to all of this and I'm trying to get an average number out of filtered rows. This is the Javascript I wrote. It is clearly wrong and hoping someone could steer me in the right direction.

`
const vehicles =$("Display Data.Rows");

let Odom = 0;

for (let i in vehicles) {Odom += vehicles[i].odometer;}

let rowLength = $("Display Data.Rows Length")

let avgOdom = Odom / rowLength;

return avgOdom;
`

**Comment by melohagan** [comment]
Hey @drearystate 

I believe Michael helped you out with this one on discord. The issue was that odometer was string type instead of number type wasn't it?

**Comment by drearystate** [comment]
Yes, that is correct.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
